integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java

```diff
@@ -111,18 +111,19 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     // install and verify operator
     installAndVerifyOperator(opNamespace, domainNamespace);
 
-    // create secret for admin credentials
+    // create secret for admin credential with special characters
+    // the resultant password is ##W%*}!"'"`']\\\\//1$$~x 
     logger.info("Create secret for admin credentials");
     String adminSecretName = "weblogic-credentials";
     assertDoesNotThrow(() -> createDomainSecret(adminSecretName, 
-            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, domainNamespace),
+            ADMIN_USERNAME_DEFAULT, "##W%*}!\"'\"`']\\\\//1$$~x", domainNamespace),
```

Comment:
create a constant in TestConstants for password with special characters and use that, instead of hard coding in all places